### ISSUE TITLE
Llenar el campo tag en developers.json

### ISSUE THREAD

**Issue by AlainPlusPlus** [indi-es/estudios#50]
Llenar el campo tag en `developers.json` de acuerdo a la siguiente definición (acrodada en https://github.com/indi-es/estudios/issues/34):
```
"type": "studio" / "individual"
"tags": ["solodev", "freelance", "indi-es"]
```

**Comment by afk-mario** [comment]
Andaba haciendo la visualización de este campo creo que tenemos que definir un poco más en donde iria la info que está en los tags, por ejemplo el Tipo de estudio ahorita lo tenemos como `individual|studio` queremos mantenerlo en su propio campo ? o mejor usamos el tag de `solo-dev` para los que son individuales ?

La ventaja de tenerlo en su propio cambo es que es más fácil hacer filter/sort, pero la ventaja de los tags es que nos permiten más libertad en la estructura y agregar mas metadatos a las cosas.

Qué opinan ?


![image](https://user-images.githubusercontent.com/1025751/115599791-9a793080-a2a1-11eb-8756-2fa470c5278f.png)

**Comment by AlainPlusPlus** [comment]
> queremos mantenerlo en su propio campo ?
Sí, a mi me parece lo más lógico pero lo puedes cambiar si quieres.
> o mejor usamos el tag de solo-dev para los que son individuales ?
La idea era de que un individuo/persona podria ser solo-dev y/o freelance. Y un estudio (que es cuando esta registrado) podría ser un solo-dev. ¿Que llevarían en el campo type los que tengan el tag de solo-dev? 

En lugar de mostrarlo como `Solo` el la web, se podría poner como `Persona`, si eso lo hace más fácil de alguna manera.

**Comment by AlainPlusPlus** [comment]
> queremos mantenerlo en su propio campo ?

    Sí, a mi me parece lo más lógico pero lo puedes cambiar si quieres.

> o mejor usamos el tag de solo-dev para los que son individuales ?

    La idea era de que un individuo/persona podria ser solo-dev y/o freelance. Y un estudio (que es cuando esta registrado) podría ser un solo-dev. ¿Que llevarían en el campo type los que tengan el tag de solo-dev?

En lugar de mostrarlo como Solo el la web, se podría poner como Persona, si eso lo hace más fácil de alguna manera.
